training/tf/tfprocess.py

```diff
@@ -278,66 +341,60 @@ def get_batchnorm_key(self):
         self.batch_norm_count += 1
         return result
 
+    def batch_norm(self, net):
+        # The weights are internal to the batchnorm layer, so apply
+        # a unique scope that we can store, and use to look them back up
+        # later on.
+        scope = self.get_batchnorm_key()
+        self.weights.append(scope + "/batch_normalization/moving_mean:0")
+        self.weights.append(scope + "/batch_normalization/moving_variance:0")
+        with tf.variable_scope(scope):
+            return tf.layers.batch_normalization(
+                    net,
+                    epsilon=1e-5, axis=1, fused=True,
+                    center=True, scale=True,
```

Comment:
You changed this and it's wrong now.

Reply:
nuts. that leaked from something else. will fix.